index.js

```diff
@@ -0,0 +1,80 @@
+module.exports = {
+	env: {
+		browser: true,
+		es6: true,
+		'jest/globals': true,
+	},
+	plugins: ['jest', 'jsdoc', 'prettier'],
+	extends: [
+		'airbnb-base',
+		'plugin:jest/recommended',
```

Comment:
Adding a note to myself: It's better to expose the jest rules separately in case projects don't use `jest` as a test runner.